requirements.txt

```diff
@@ -1,28 +1,33 @@
-# The order of packages is significant, because pip processes them in the order
```

Comment:
We should get rid of requirements.txt like in upstream tempest 
It always introduced maintenance problems